stories/browser/src/UIKeySheet.tsx

```diff
@@ -2,19 +2,31 @@ import * as React from 'react';
 import { View, StyleSheet } from 'react-native';
 
 import { UIKeyTextView, UIMaterialTextViewRef } from '@tonlabs/uikit.inputs';
+import { UIBoxButton } from '@tonlabs/uikit.controls';
 
 import { UIPullerSheet } from './UIPullerSheet';
 
+function useKeySheetContent() {
+    const [error, setError] = React.useState(false);
+    const [success, setSuccess] = React.useState(false);
+    const [key, setKey] = React.useState('');
```

Comment:
You can use `useRef` here instead, as the change of it doesn't update any UI, so there is no need to trigger re-render here

Reply:
https://github.com/tonlabs/UIKit/pull/1476/commits/9a0447c5cc35417b0ecb3622a1aaebf571c8e06b